### PR TITLE
Feat: Adds `RadioGroup` component

### PR DESCRIPTION
**Pull request by eduardoformiga**
## What's the purpose of this pull request?

- [x] Creates `RadioGroup` components on @faststore/components
- [x] Adapts `RadioGroup` on @faststore/ui
- [x] Adds `RadioGroup` documentation page on Nextra

## How to test it?

run `yarn dev` in app/site folder and checks the `RadioGroup` component or test it using the doc [preview link](https://faststore-site-jyqbhsk0s-faststore.vercel.app/components/molecules/radio-group).

### Starters Deploy Preview

We don't use `RadioGroup` in our starter, but we use `RadioOption`, so you can see the [SkuSelector](https://nextjs-store-storybook-jhup6bz9i-faststore.vercel.app/?path=/docs/molecules-skuselector-overview--overview-default-1) preview page.

[Feat: Replace RadioGroup FSUI ](https://github.com/vtex-sites/nextjs.store/pull/345)

### CODE REVIEW

File: packages/components/src/molecules/RadioGroup/RadioOption.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ export interface RadioOptionProps
 
 const RadioOption = forwardRef<HTMLInputElement, RadioOptionProps>(
   function RadioOption(
-    { label, value, children, testId = 'store-radio-option', ...otherProps },
+    { label, value, children, testId = 'fs-radio-option', ...otherProps },
```
Comment (hellofanny) [inline packages/components/src/molecules/RadioGroup/RadioOption.tsx:26]: @eduardoformiga do you think we can rename it to be more specific to this context?

something like `fs-radio-group-option`

And I think we can also rename the data-attr of this component.
We are currently using `data-fs-radio-option`.

If you do so, remember to add this data-attr, to the `Customization` section. 😉

File: apps/site/pages/components/molecules/radio-group.mdx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    </OverviewSection>
+  </Tab>
+  <Tab>
+    ```tsx
+    <RadioGroup name="group">
+      <RadioField label="First option" name="group" id="radiofield-first" />
+      <RadioField label="Second option" name="group" id="radiofield-second" />
+      <RadioField label="Third option" name="group" id="radiofield-third" />
+    </RadioGroup>
+    ```
```
Comment (hellofanny) [inline apps/site/pages/components/molecules/radio-group.mdx:40]:
```suggestion
<Tabs items={['Example', 'Code']} defaultIndex="0">
  <Tab>
    <OverviewSection>
      <RadioGroup name="radiogroup">
        <RadioField label="First option" name="group" id="radiogroup-first" />
        <RadioField label="Second option" name="radiogroup" id="radiogroup-second" />
        <RadioField label="Third option" name="radiogroup" id="radiogroup-third" />
      </RadioGroup>
    </OverviewSection>
  </Tab>
  <Tab>
    ```tsx
    <RadioGroup name="radiogroup">
      <RadioField label="First option" name="radiogroup" id="radiogroup-first" />
      <RadioField label="Second option" name="radiogroup" id="radiogroup-second" />
      <RadioField label="Third option" name="radiogroup" id="radiogroup-third" />
    </RadioGroup>
    ```
```

**Comment by eduardoformiga** [comment]
> Nice jobbbb @eduardoformiga!! 🚀 I've left a few suggestion, can you take a look pls 😉

thank you, @hellofanny . Just notice that we have `RadioOption` and `RadioField`. I'll refactor this component doc to use RadioOption instead.

File: apps/site/pages/components/molecules/radio-group.mdx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  </RadioGroup>
+</OverviewSection>
+
+```tsx
+<RadioGroup name="group">
+  <RadioField label="First option" name="group" id="usage-radiofield-first" />
+  <RadioField label="Second option" name="group" id="usage-radiofield-second" />
+  <RadioField label="Third option" name="group" id="usage-radiofield-third" />
+</RadioGroup>
+```
```
[the remark this answers is not in the document]
Comment (eduardoformiga) [reply apps/site/pages/components/molecules/radio-group.mdx]: done [here](https://github.com/vtex/faststore/pull/1604/commits/f712cf95d1df2344148f62f68a91d356a9b75adf)

File: packages/components/src/molecules/RadioGroup/RadioOption.tsx (hunk 1 above)
Comment (eduardoformiga) [reply packages/components/src/molecules/RadioGroup/RadioOption.tsx]: Nice! done [here](https://github.com/vtex/faststore/pull/1604/commits/497c062a099441df64a5efc8cf88a2afb659299f).

**Review by emersonlaurentino — APPROVED** [review]

**Review by hellofanny — APPROVED** [review]

**Pull request merged**